bin/gpo

```diff
@@ -404,8 +405,10 @@ class gPodderCli(object):
 
             return '   '
 
-        episodes = ('%3d.%s %s %s' % (i + 1, ((' %s' % e.guid) if show_guid else ''), status_str(e), e.title)
-                for i, e in enumerate(podcast.get_all_episodes()))
+        episodes = ('%3d.%s %s %s' % (i + 1, ((' %s' % e.guid) if
+                                              show_guid else ''),
+                                      status_str(e), e.title)
+                    for i, e in enumerate(podcast.get_all_episodes()))
```

Comment:
this is not readable. I'd put the string formatting of one episode in a local function, with the guid part in a local variable in it.